scrapers/AssTraffic.yml

```diff
@@ -0,0 +1,60 @@
+name: "AssTraffic"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - asstraffic.com/movies/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - asstraffic.com/models/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h2
+      Date:
+        selector: /html[1]/body[1]/div[1]/div[1]/div[1]/div[1]/div[2]/span[1]//text()
+        replace:
+          - regex: "Added "
+            with:
+        parseDate: January 2, 2006
+      Details: //p[@class="mg-md"]//text()
+      Performers:
+        Name: //div[@id="video-info"]//a[contains(@href,'/models')]/text()
+      Studio:
+        Name:
+          Fixed: AssTraffic
+      Image: //video[@id='video']/@poster
+
```

Comment:
tags can also be added for the scene scraper
```yaml
      Tags:
        Name: //div[@class="col-sm-8 col-md-8 no-padding-side tag-container"]/a/text()
```

Reply:
The reason why I didn't do this initially was that AssTraffic uses dictionaries for tags in contrast to our flat list model. This will result in garbage tags sometimes with `big`, `average` etc values for the various body related tags. But there are many tags that make sense on their own, so it's better than nothing.